cmd/neofs-cli/docs/storage-node-xheaders.md

```diff
@@ -0,0 +1,29 @@
+# Extended headers
+
+## Overview
+
+Extended headers are used for Request/Response. They may contain any user-defined headers
+to be interpreted on application level.
+Key name must be a unique valid UTF-8 string. Value can't be empty. Requests or
+Responses with duplicated header names or headers with empty values will be
+considered invalid.
+
+## Existing headers
+
+There are some "well-known" headers starting with `__NEOFS__` prefix that
+affect system behaviour:
+
+* `__NEOFS__NETMAP_EPOCH` - netmap epoch to use for object placement calculation. The `value` is string
+encoded `uint64` in decimal presentation. If set to '0' or not set, the
+current epoch only will be used.
+* `__NEOFS__NETMAP_LOOKUP_DEPTH` - if object can't be found using current epoch's netmap, this header limits
+how many past epochs the node can look up through. The `value` is string
+encoded `uint64` in decimal presentation. If set to '0' or not set, only the
+current epoch will be used.
+
+## `neofs-cli` commands with `--xhdr`
+
+List of commands with support of extended headers:
```

Comment:
I would like to suggest to add at some examples of xheaders usage, but it's up to you.

Reply:
Added.